ngraph/test/onnx/onnx_import.in.cpp

```diff
@@ -2928,6 +2928,20 @@ NGRAPH_TEST(${BACKEND_NAME}, onnx_model_instance_normalization)
     test_case.run();
 }
 
+NGRAPH_TEST(${BACKEND_NAME}, onnx_instance_normalization_dynamic)
+{
+    auto function = onnx_import::import_onnx_model(
+        file_util::path_join(SERIALIZED_ZOO, "onnx/instance_norm_dynamic.prototxt"));
+
+    auto test_case = test::TestCase<TestEngine, test::TestCaseType::DYNAMIC>(function);
+    std::vector<float> input_data(3);
+    std::iota(std::begin(input_data), std::end(input_data), 1.0f);
```

Comment:
```suggestion
    std::vector<float> input_data{1.f, 2.f, 3.f};
```
I think that iota for 3 (known upfront) elements is an overhead. Use initializer list.